docs/SOPs/archiving_SOP.md

```diff
@@ -370,9 +367,3 @@ Once the files are uploaded and validated, to finish the submission, just run th
 ```
 curl -X POST https://archiver.ingest.staging.archive.data.humancellatlas.org/archiveSubmissions/<dsp_submission_uuid>/complete -H 'Api-Key: {archiver_api_key}' 
 ```
-
-## Triggering re-archival of submissions
-
-
-To trigger the re-archival of submissions, please refer to the proper section in the [Ingestion to archives](https://docs.google.com/document/d/1S4fyCSqB3nLrCUssNMwSp6ff8tmeipMi_slnXW2Lrq4/edit?pli=1#heading=h.wzojhbhfywqr) document
-
```

Comment:
Removing this for now. I think we don't need to do those steps in case something goes wrong.